server/routes/findUser.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../models/User");
+
+router.get("/:email", async (req, res, next) => {
```

Comment:
For getting a user it's better to use something immutable like id. You could also consolidate the find and delete routes into one userRouter to improve readability

Reply:
Thanks for the encouragement.  Updates completed.